Description: force the ullong conversion anywhere.
Author: Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>
Index: hdf5/src/H5Tconv.c
===================================================================
--- hdf5.orig/src/H5Tconv.c
+++ hdf5/src/H5Tconv.c
@@ -7475,14 +7475,12 @@ H5T__conv_ullong_double(hid_t src_id, hi
  *
  *-------------------------------------------------------------------------
  */
-#ifdef H5T_CONV_INTERNAL_ULLONG_LDOUBLE
 herr_t
 H5T__conv_ullong_ldouble(hid_t src_id, hid_t dst_id, H5T_cdata_t *cdata, size_t nelmts, size_t buf_stride,
                          size_t H5_ATTR_UNUSED bkg_stride, void *buf, void H5_ATTR_UNUSED *bkg)
 {
     H5T_CONV_xF(ULLONG, LDOUBLE, unsigned long long, long double, -, -);
 }
-#endif /*H5T_CONV_INTERNAL_ULLONG_LDOUBLE*/
 
 /*-------------------------------------------------------------------------
  * Function:    H5T__conv_float_schar
